### PR TITLE
CHANGELOG: update release note for 2.0.2

### PR DESCRIPTION
**Pull request by gozssky**
### What problem does this PR solve? 

update release note for 2.0.2

### Check List 

Tests 
 - No code

### CODE REVIEW

**Comment by lance6716** [comment]
/lgtm

wish we could release today

**Comment by lance6716** [comment]
I plan to add a "known issues" https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2 in release note, please help add that

**Comment by lance6716** [comment]
I plan to add a "known issues" https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2 in release note, please help add that

**Comment by gozssky** [comment]
> I plan to add a "known issues" https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2 in release note, please help add that

Do we need to list issues one by one? Or just this link?

**Comment by lance6716** [comment]
> > I plan to add a "known issues" https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2 in release note, please help add that
> 
> @lance6716 Do we need to list issues one by one? Or just this link?

I think a link is OK.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,46 @@
...
+- Improve the stability of DM automatically getting the SQL mode from the binlog file [#1382](https://github.com/pingcap/dm/pull/1382) [#1552](https://github.com/pingcap/dm/pull/1552)
+- Support automatically parsing GTIDs of different formats in the source configuration file [#1385](https://github.com/pingcap/dm/pull/1385)
+- Extend DM-worker’s TTL for keepalive to reduce scheduling caused by poor network [#1405](https://github.com/pingcap/dm/pull/1405)
+- Support reporting an error when the configuration file contains configuration items that are not referenced [#1410](https://github.com/pingcap/dm/pull/1410)
+- Improve the display of a GTID set by sorting it in dictionary order [#1424](https://github.com/pingcap/dm/pull/1424)
+- Optimize monitoring and alerting rules [#1438](https://github.com/pingcap/dm/pull/1438)
+- Support manually transferring an upstream data source to a specified DM-worker [#1492](https://github.com/pingcap/dm/pull/1492)
+- Add configurations of etcd compaction and disk quota [#1521](https://github.com/pingcap/dm/pull/1521)
+
+### Bug fixes
```
Comment (lance6716) [inline CHANGELOG.md:31]:
```suggestion
### Bug fixes

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,46 @@
...
+- Fix the issue that enabling the relay log feature might cause data loss when the GTIDs specified in upstream binlog `previous_gtids` events are not consecutive [#1390](https://github.com/pingcap/dm/pull/1390) [#1430](https://github.com/pingcap/dm/pull/1430)
+- Disable the heartbeat feature of DM v1.0 to avoid the failure of high availability scheduling [#1467](https://github.com/pingcap/dm/pull/1467)
+- Fix the issue that the migration fails if the upstream binlog sequence number is larger than 999999 [#1476](https://github.com/pingcap/dm/pull/1476)
+- Fix the issue that DM commands hang when DM gets stuck in pinging the upstream and downstream databases [#1477](https://github.com/pingcap/dm/pull/1477)
+- Fix the issue that the full import fails when the upstream database enables the `ANSI_QUOTES` mode [#1497](https://github.com/pingcap/dm/pull/1497)
+- Fix the issue that DM might duplicate binlog events when the GTID and the relay log are enabled at the same time [#1525](https://github.com/pingcap/dm/pull/1525)
+
+### Known issues
```
Comment (lance6716) [inline CHANGELOG.md:42]:
```suggestion
### Known issues

```

**Review by lance6716 — CHANGES_REQUESTED** [review]

**Review by lance6716 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,48 @@
...
+
+https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2
```
Comment (lance6716) [inline CHANGELOG.md:45]:
```suggestion
[GitHub issues](https://github.com/pingcap/dm/issues?q=is%3Aissue+is%3Aopen+label%3Aaffected-v2.0.2)
```

**Pull request merged**